code/backend/batch/utilities/tools/QuestionAnswerTool.py

```diff
@@ -1,182 +1,113 @@
 import json
 import logging
+from typing import List
 import warnings
 from .AnsweringToolBase import AnsweringToolBase
 
-from langchain.chains.llm import LLMChain
-from langchain.prompts import (
-    AIMessagePromptTemplate,
-    ChatPromptTemplate,
-    FewShotChatMessagePromptTemplate,
-    HumanMessagePromptTemplate,
-    MessagesPlaceholder,
-    PromptTemplate,
-)
-from langchain_community.callbacks import get_openai_callback
-from langchain_core.documents import Document
-from langchain_core.messages import SystemMessage
-
 from ..helpers.AzureSearchHelper import AzureSearchHelper
 from ..helpers.ConfigHelper import ConfigHelper
 from ..helpers.LLMHelper import LLMHelper
 from ..helpers.EnvHelper import EnvHelper
 from ..common.Answer import Answer
 from ..common.SourceDocument import SourceDocument
+from azure.search.documents.models import VectorizedQuery
+
 
 logger = logging.getLogger(__name__)
 
 
 class QuestionAnswerTool(AnsweringToolBase):
+
     def __init__(self) -> None:
         self.name = "QuestionAnswer"
-        self.vector_store = AzureSearchHelper().get_vector_store()
-        self.verbose = True
+        self.azure_search_helper = AzureSearchHelper()
+        self.llm_helper = LLMHelper()
```

Comment:
Assuming that #767 is going to conflict here

Reply:
Yes it is, I'll let that go in first before making changes for real.